README.md

```diff
@@ -25,6 +25,21 @@
 
 **Blog App** The Blog app is classic example of a blog website. A fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
```

Comment:
- [ ] Kindly add .vscode to your gitignore as it makes project unclean.

Reply:
It's ready. Thank you.